pkg/daemon/ceph/client/upgrade.go

```diff
@@ -27,6 +27,10 @@ import (
 	"github.com/rook/rook/pkg/util"
 )
 
+const (
+	maxRetries = 10
```

Comment:
If we have the same number of retries for different intervals, the individual retry interval will vary a lot. How about we calculate the number of retries based on retrying every 10s?

Reply:
That's a good point. Keeping retry delay to 10s and calculating maxRetries based on the `waitTimeoutForHealthyOSD` set by user. Now:
retries = waitTimeoutForHealthyOSD(in seconds) / 10
delay = 10s